### PR TITLE
fix validation accuracy problem

### LINKED ISSUE #75: Incorrect validation accuracy when size of validation set is not divisible by batch_size
Hi,

Thank you so much for providing us such a useful framework.

I recently find a problem.

In https://github.com/tflearn/tflearn/blob/master/tflearn/helpers/trainer.py#L801, the function evaluating validation loss and accuracy, there might be a problem when size of validation set is not divisble by batch_size.

```{python}
def evaluate_flow(session, ops_to_evaluate, dataflow):
        if not isinstance(ops_to_evaluate, list):
            ops_to_evaluate = [ops_to_evaluate]
        tflearn.is_training(False, session)
        dataflow.reset()
        dataflow.start()
        res = [0. for i in ops_to_evaluate]
        feed_batch = dataflow.next()
        n_batches = len(dataflow.batches)
        while feed_batch:
            r = session.run(ops_to_evaluate, feed_batch)
            for i in range(len(r)):
                res[i] += r[i] / n_batches # BUG here
            feed_batch = dataflow.next()
        return res
```

Could I make a pull request to fix the problem? Thank you!

### PR DESCRIPTION
**Pull request by yzgysjr**
fix #75

### CODE REVIEW

**Comment by aymericdamien** [comment]
Thanks for the fix!
